cloudinit/config/cc_resolv_conf.py

```diff
@@ -108,18 +108,19 @@ def handle(name, cfg, cloud, log, _args):
 
     if "resolv_conf" not in cfg:
         log.warning("manage_resolv_conf True but no parameters provided!")
+        return
 
     try:
         template_fn = cloud.get_template_filename(
-            RESOLVE_CONFIG_TEMPLATE_MAP[cloud.distro.resolv_conf_fn])
+            RESOLVE_CONFIG_TEMPLATE_MAP[cloud.distro.resolve_conf_fn])
```

Comment:
Another typo. I don't know how, luckily I saw this file for no reason. And found these blunders